templates/html-header.twig

```diff
@@ -85,10 +85,17 @@
         </script>
 	{% endif %}
 
-	<link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+	<link href="https://fonts.googleapis.com/css?family=Lora|Roboto" rel="stylesheet">
 	<link rel="pingback" href="{{ site.pingback_url }}">
```

Comment:
This line is probably redundant and should be completely removed. We already have these fonts on [scss code](https://github.com/greenpeace/planet4-master-theme/blob/develop/assets/scss/base/_fonts.scss).